hrjobcontract/js/src/job-contract/controllers/form/form-leave.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'job-contract/vendor/fraction',
```

Comment:
It doesn't seem you're using this vendor at all. If that's not the case, please remove it from this module and maybe check also if it's used at all in the entire extension, and if not just delete it altogether